roles/qe_common/README.md

```diff
@@ -0,0 +1,42 @@
+telemetry_logging
```

Comment:
This file needs a re-write so it refers to the correct role;

In think that this should be renamed as well.
common_helper_tasks, for example would be more descriptive.

Reply:
Done